app/packages/looker/src/elements/imavid/index.ts

```diff
@@ -227,7 +230,7 @@ export class ImaVidElement extends BaseElement<ImaVidState, HTMLImageElement> {
     // if abs(frameNumberToDraw, currentFrameNumber) > 1, then skip
     // this is to avoid drawing frames that are too far apart
     // this can happen when user is scrubbing through the video
-    if (Math.abs(frameNumberToDraw - this.frameNumber) > 1) {
+    if (Math.abs(frameNumberToDraw - this.frameNumber) > 1 && !this.isLoop) {
```

Comment:
this seems to fix the looping issue. For some reason when looping at lower speeds specifically this method would get triggered causing it to "buffer" and then try showing a frame again. This would cause that weird jumping between 2 and 3. Disabling this functionality while looping seems to fix the problem without any side-effects from what I can tell testing locally.